frontend/src/components/Tools.js

```diff
@@ -15,22 +15,22 @@ function Tools() {
             <div className="container-fluid">
                 <Row >
                     <Col>
```

Comment:
The app wouldn't run when I tried npm start because the images didn't have an "alt" description so I included that to make it work.